make/kind_images.sh

```diff
@@ -18,6 +18,7 @@ KIND_IMAGE_K8S_120=docker.io/kindest/node@sha256:6f2d011dffe182bad80b85f6c00e8ca
 KIND_IMAGE_K8S_121=docker.io/kindest/node@sha256:f316b33dd88f8196379f38feb80545ef3ed44d9197dca1bfd48bcb1583210207
```

Comment:
suggestion: I think `./devel/cluster/kind_cluster_node_versions.sh` and `./make/kind_images.sh` should be identical and I don't think they are in this PR?

Did you run `make latest-kind-images` after changing `./hack/latest-kind-images.sh`?